### PR TITLE
Hide tray for teachers [test ui]

### PR DESCRIPTION
**Pull request by mehalshah**

### CODE REVIEW

File: dashboard/app/models/script.rb [hunk 1]
```diff
@@ -353,6 +353,10 @@ def freeplay_links
 
   end
 
+  def professional_course?
+    pd? || professional_learning_course?
```
Comment (aoby) [inline dashboard/app/models/script.rb:357]: Where are these defined?

File: dashboard/test/controllers/script_levels_controller_test.rb [hunk 2]
```diff
@@ -830,14 +830,26 @@ def assert_caching_enabled(cache_control_header, max_age, proxy_max_age)
     assert_equal nil, assigns(:user)
   end
 
-  test 'does not show teacher panel for pd scripts' do
+  test 'foobar' do
```
Comment (aoby) [inline dashboard/test/controllers/script_levels_controller_test.rb]: as discussed offline, rename this test before merging

File: dashboard/app/models/script.rb (hunk 1 above)
Comment (mehalshah) [inline dashboard/app/models/script.rb:357]: Script.rb. I think they are autogenerated from serialized attributes

**Comment by aoby** [comment]
lgtm

**Pull request merged**